src/std/string.md

```diff
@@ -27,8 +27,7 @@ fn main() {
 
 <details>
 
-* `len` returns the size of the `String` in bytes, not its length in characters.
-* `chars` returns an iterator over the actual characters.
-* `String` implements `Deref<Target = str>` which transparently gives it access to `str`'s methods.
```

Comment:
Why remove this line?

Reply:
A very similar statement with the same information is already written on the slide itself, it seemed redundant to repeat it again in speaker notes. The one in the slide also links to the appropriate place in the STD reference.